utils/generate-testmodels.py

```diff
@@ -1,9 +1,12 @@
+import shutil
 from pathlib import Path
 
 import ribasim_testmodels
 
 if __name__ == "__main__":
     datadir = Path("data")
+    if datadir.is_dir():
+        shutil.rmtree(datadir)
```

Comment:
Oh this would delete a lot of useful stuff for me. I could move my stuff to another folder and gitignore that. Good to announce this change though.

Reply:
I agree it's nice to have a clean folder. Perhaps having a more dedicated name for it, like testmodels, perhaps with a README explaining it's use and that it is automatically emptied, would be better.

Reply:
Sounds good to me.

Reply:
Done in https://github.com/Deltares/Ribasim/pull/604/commits/91590e09c08037f9c360579682c63ef8974fc5dd